test/integration/etcd/server.go

```diff
@@ -347,25 +351,25 @@ func CreateTestCRDs(t *testing.T, client apiextensionsclientset.Interface, skipC
 }
 
 func createTestCRD(t *testing.T, client apiextensionsclientset.Interface, skipCrdExistsInDiscovery bool, crd *apiextensionsv1beta1.CustomResourceDefinition) {
-	if _, err := client.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd); err != nil {
-		t.Fatalf("Failed to create %s CRD; %v", crd.Name, err)
+	if _, err := client.ApiextensionsV1beta1().CustomResourceDefinitionsWithMultiTenancy(crd.Tenant).Create(crd); err != nil {
+		t.Fatalf("Failed to create %s CRD under tenant %v : %v", crd.Name, crd.Tenant, err)
 	}
 	if skipCrdExistsInDiscovery {
-		if err := waitForEstablishedCRD(client, crd.Name); err != nil {
+		if err := waitForEstablishedCRD(client, crd); err != nil {
 			t.Fatalf("Failed to establish %s CRD; %v", crd.Name, err)
 		}
 		return
 	}
-	if err := wait.PollImmediate(500*time.Millisecond, wait.ForeverTestTimeout, func() (bool, error) {
+	if err := wait.PollImmediate(500*time.Millisecond, time.Second*2, func() (bool, error) {
```

Comment:
extract time.Second*2 into a variable to be share with other places

Reply:
Accidental change. Reverting the change.